collect_app/src/main/java/org/odk/collect/android/map/WebMapService.java

```diff
@@ -0,0 +1,65 @@
+package org.odk.collect.android.map;
+
+import org.odk.collect.android.application.Collect;
+import org.osmdroid.tileprovider.MapTile;
+import org.osmdroid.tileprovider.tilesource.OnlineTileSourceBase;
+
+import java.io.Serializable;
+
+/**
+ * A serializable definition of a Web Map Service in terms of its URL structure
+ * and the parameters for fetching tiles from it.
+ */
+public class WebMapService implements Serializable {
+    public final String cacheName;
+    public final int minZoomLevel;
+    public final int maxZoomLevel;
+    public final int tileSize;
+    public final String copyright;
+    public final String[] urlTemplates;
+    public OnlineTileSourceBase onlineTileSource = null;
+
+    public WebMapService(String cacheName, int minZoomLevel, int maxZoomLevel,
+        int tileSize, String copyright, String... urlTemplates) {
+        this.cacheName = cacheName;
+        this.minZoomLevel = minZoomLevel;
+        this.maxZoomLevel = maxZoomLevel;
+        this.tileSize = tileSize;
+        this.copyright = copyright;
+        this.urlTemplates = urlTemplates;
+    }
+
+    @Deprecated public WebMapService(int cacheNameStringId, int minZoomLevel,
```

Comment:
This is what we'll get rid of when we're ready to stop translating the cache names.